src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -178,6 +178,9 @@ def handle_message(self, message):
                     "timestamp": contents["timestamp"],
                     "query_subscription_id": contents["subscription_id"],
                     "contents": contents,
+                    "offset": message.offset(),
+                    "partition": message.partition(),
+                    "value": message.value(),
```

Comment:
Isn't this redundant with `contents`? Or are you just trying to isolate that it isn't an issue with `parse_message_value`?

Reply:
It is, but I figure that it could potentially be different at some point so seems worth logging.